components/server/src/server.ts

```diff
@@ -98,7 +102,20 @@ export class Server<C extends GitpodClient, S extends GitpodServer> {
     protected installationAdminHttpServer?: http.Server;
     protected disposables = new DisposableCollection();
 
+    public async testLoop() {
```

Comment:
Should be reverted before merging?

Reply:
> Should be reverted before merging?

[]()

Reply:
> yes

[]()

Reply:
- [ ] GAXZC5ZV5UWV3ZL4GPCIH24PXYURREXQGOWHVPD57DIW3EPXIOQJE5WV